lib/timeline/Core.js

```diff
@@ -1003,7 +1003,6 @@ Core.prototype._redraw = function() {
   } else {
     this.redrawCount = 0;
   }
-  this.initialDrawDone = true;
```

Comment:
Doesn't this need to be set? It's the only place in the module where it goes to `true`.

OK, got it. It's set in `Timeline._activateOnInitialDrawDone()`. I find it a bit confusing that this is in a derived class.

Reply:
`Timeline._activateOnInitialDrawDone()` was incorrect. I removed it.